src/liballoc/rc.rs

```diff
@@ -1179,6 +1179,14 @@ impl<T> From<Vec<T>> for Rc<[T]> {
     }
 }
 
+#[unstable(feature = "rc_into_nonnull", reason = "newly added", issue = "0")]
```

Comment:
We don't currently support unstable impls so this would be stable immediately. Please change to a `stable` attribute. Same for Arc.